plan_test.go

```diff
@@ -0,0 +1,71 @@
+package plan
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform/terraform"
+	"os"
+	"testing"
+)
+
+func OpenReadPlan(testCase string) *terraform.Plan {
+	file, err := os.Open(testCase)
+	if err != nil {
+		fmt.Errorf("unexpected error %v", err)
```

Comment:
This error is going no where. You might want to panic or you pass in the `*testing.T`  and use it's Fatalf